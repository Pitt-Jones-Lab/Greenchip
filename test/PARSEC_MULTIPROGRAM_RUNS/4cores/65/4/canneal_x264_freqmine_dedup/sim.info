{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/canneal_x264_freqmine_dedup',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_65_4_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/canneal_x264_freqmine_dedup',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpeNFodl/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb20',
 'pin_version': 'unknown',
 'rusage': (36661.417616999999,
            4126.4116899999999,
            1169096,
            0,
            0,
            0,
            289282,
            0,
            0,
            8,
            1896,
            0,
            0,
            0,
            345239743,
            10378854),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/canneal_x264_freqmine_dedup --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem65.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_65_4_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpeNFodl/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/65/4/canneal_x264_freqmine_dedup/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 18851.599202871323,
 't_start': 1463345288.635128,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 2145841152L}
