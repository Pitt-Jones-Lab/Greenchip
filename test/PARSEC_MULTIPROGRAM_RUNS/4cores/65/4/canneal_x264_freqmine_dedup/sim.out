                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |   985163500 |  3553369884 |  3061903364 |  2398423751
  Cycles                           |  2728711524 |  2728711524 |  2728711524 |  2728711524
  IPC                              |        0.36 |        1.30 |        1.12 |        0.88
  Time (ns)                        |  1025831400 |  1025831400 |  1025831400 |  1025831400
  Idle time (ns)                   |       19897 |        8613 |      927078 |       37905
  Idle time (%)                    |        0.0% |        0.0% |        0.1% |        0.0%
Branch predictor stats             |             |             |             |            
  num correct                      |   112222639 |    97633790 |   612689072 |    70249900
  num incorrect                    |     6689044 |    14746457 |    19916877 |     6476734
  misprediction rate               |       5.63% |      13.12% |       3.15% |       8.44%
  mpki                             |        6.79 |        4.15 |        6.50 |        2.70
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   145320817 |   365434315 |   448908079 |   242328238
    num misses                     |         189 |        1961 |         120 |        3442
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |   425643086 |  1303766832 |  1805554136 |  1084128501
    num misses                     |     2335214 |    12122541 |      569028 |    10343571
    miss rate                      |       0.55% |       0.93% |       0.03% |       0.95%
    mpki                           |        2.37 |        3.41 |        0.19 |        4.31
  L2 TLB                           |             |             |             |            
    num accesses                   |     2335403 |    12124502 |      569148 |    10347013
    num misses                     |      948193 |     4993188 |       65833 |     4123208
    miss rate                      |      40.60% |      41.18% |      11.57% |      39.85%
    mpki                           |        0.96 |        1.41 |        0.02 |        1.72
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   145320817 |   365434315 |   448908079 |   242328238
    num cache misses               |     1074729 |    28574218 |       37997 |    21919473
    miss rate                      |       0.74% |       7.82% |       0.01% |       9.05%
    mpki                           |        1.09 |        8.04 |        0.01 |        9.14
  Cache L1-D                       |             |             |             |            
    num cache accesses             |   425643086 |  1303766832 |  1805554136 |  1084128501
    num cache misses               |    10888280 |    36765559 |     4310677 |    70228200
    miss rate                      |       2.56% |       2.82% |       0.24% |       6.48%
    mpki                           |       11.05 |       10.35 |        1.41 |       29.28
  Cache L2                         |             |             |             |            
    num cache accesses             |    15442463 |    65590160 |     5372859 |    97130736
    num cache misses               |     9153103 |     9660333 |     1852700 |    51039907
    miss rate                      |      59.27% |      14.73% |      34.48% |      52.55%
    mpki                           |        9.29 |        2.72 |        0.61 |       21.28
DRAM summary                       |             |             |             |            
  num dram accesses                |   111406525 |           0 |           0 |           0
  average dram access latency (ns) |      187.93 |         inf |         inf |         inf
  average dram queueing delay      |      187.93 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     5091144 |     7953945 |     1530894 |    27713018
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
